include/unicorn/arm.h

```diff
@@ -211,6 +211,7 @@ typedef enum uc_arm_reg {
     UC_ARM_REG_XPSR_NZCVQ,
     UC_ARM_REG_XPSR_G,
     UC_ARM_REG_XPSR_NZCVQG,
+    UC_ARM_REG_ESR,
```

Comment:
We shall always append enums or the values of some enums like `UC_ARM_REG_CP_REG` will be changed.

Also please add a small comment here to indicate it is a pseudo register.

Reply:
done